sdk/monitor/azquery/CHANGELOG.md

```diff
@@ -3,8 +3,12 @@
 ## 1.2.0-beta.2 (Unreleased)
```

Comment:
Is the plan to ship v1.0 this month?

Reply:
Planning to GA end of January or February. Was going to update the version when prepping for release, but can do it.